models/issue.go

```diff
@@ -1171,6 +1171,10 @@ func newIssue(e *xorm.Session, doer *User, opts NewIssueOptions) (err error) {
 		}
 
 		for i := 0; i < len(attachments); i++ {
+			if !attachments[i].IsNotAttached() {
```

Comment:
It looks like `if attachments[i].IsAttached() {` should be better.

Reply:
Yes and re-thinking about it I will use the term IsLinked to not reuse the term attach